addons/rmsmartshape/plugin-functionality.gd

```diff
@@ -53,41 +53,41 @@ static func _get_intersecting_control_point(
 static func action_set_pivot(
 	update_node: Node,
 	update_method: String,
-	undo: UndoRedo,
+	undo: EditorUndoRedoManager,
```

Comment:
Since this argument in all `action_*` functions is always `get_undo_redo()`, it can be removed completely.

Reply:
Do you mean remove the argument from all `action_*` functions? We need to pass the reference - there's no other way to get it but from `EditorPlugin.get_undo_redo()`

Reply:
Ah ok, I thought it was global. Then nevermind.